File: practice/programiz/The_Number_Is_Even_Odd.c
#include<stdio.h>
/***
 * @brief in this program we check number is even or odd by teaking thourgh user input
 * @return int 
 * @implements number%2 == 0 ; this satements is aplicable for even number
 */
int main()
{   
    int num;
    printf("Enter the Number is: ");
    scanf("%i",&num);
    if (num%2 == 0)
    {
        printf("the number is even %i",num);

    }
    else
    {
        printf("the number is odd %i",num);
    }
    return 0;
}

File: practice/java/TheNumberIsEvenOdd.java
import java.util.Scanner;


class EvenOdd{
    public static void main(String args[])

    {
       int num;
      ;
      Scanner input = new Scanner(System.in);
       System.out.println("Enter a  Number:\n");  

       num = input.nextInt();
       if(num%2 ==0)
       {
        System.out.println("The Number Is Even:"+num);
       }
       else {
         System.out.println("The Number Is Odd:"+num);
       }

       
    }
}